src/js/chatWindow.js

```diff
@@ -1,8 +1,20 @@
-export const addMessageToChatHistory = (m) => {
+const guardOptions = (options) => {
+  if (options.messageType !== 'outgoing' && options.messageType !== 'incoming') {
+    return false;
+  }
+
+  return true;
+};
+
+export const addMessageToChatHistory = (m, options = { messageType: 'outgoing' }) => {
   if (typeof m !== 'string' || m === '') return;
 
+  if (!guardOptions(options)) throw new Error('Message Type is incorrect');
+
   const c = document.createElement('div');
-  c.classList.add('chat-window__message', 'chat-window__message--outgoing');
+
+  c.classList.add('chat-window__message', `chat-window__message--${options.messageType}`);
```

Comment:
is there a more descriptive variable that we can use for `c`?

Reply:
Yep! I changed it to chatMessageElement